src/guides/v2.4/release-notes/release-notes-2-4-1-commerce.md

```diff
@@ -0,0 +1,1123 @@
+
+<!--- ENGCOM-7285-->
+
+<!--- ENGCOM-7631-->
+
+<!--- ENGCOM-7010-->
+
+<!--- ENGCOM-7244-->
+
+<!--- ENGCOM-7770-->
+
+<!--- ENGCOM-7619-->
+
+<!--- ENGCOM-7873-->
+
+<!--- ENGCOM-7639-->
+
+<!--- ENGCOM-7156-->
+
+<!--- ENGCOM-7981-->
+
+<!--- ENGCOM-7911-->
+
+<!--- ENGCOM-8007-->
+
+<!--- ENGCOM-7992-->
+
+<!--- ENGCOM-7967-->
+
+<!--- ENGCOM-8015-->
+
+### URL rewrites
+
+<!--- MC-34483-->
+
+*  You can now successfully preview the staging schedule for a CMS block. Previously, Magento threw a fatal error.
+
+<!--- MC-33028-->
+
+*  Magento now preserves existing catalog URL rewrites as expected when a store view is assigned to a different store. Previously, Magento deleted the store-specific URL rewrites.
+
+### Vault
+
+<!--- MC-34674-->
+
+*  Magento no longer saves credit card numbers when the **Save for later use** checkbox on the payment section of the checkout workflow is not selected.
+
+### Visual Merchandiser
+
+<!--- MC-35574-->
+
+*  Magento now displays source stock instead of the default product stock when you sort products in Visual Merchandiser and Inventory is enabled.
+
+### Web API framework
+
+<!--- ENGCOM-7618-->
+
+<!--- ENGCOM-7709-->
+
+<!--- ENGCOM-7665-->
+
+<!--- ENGCOM-7612-->
+
+<!--- ENGCOM-7785-->
+
+<!--- ENGCOM-7715-->
+
+<!--- ENGCOM-7829-->
+
+<!--- ENGCOM-7611-->
+
+<!--- MC-36419-->
+
+<!--- MC-36084-->
+
+*  Invoices created using REST now include gift card information similar to how as invoices created in the Admin do. Previously, using POST `\{host}/rest/default/V1/order/3/invoice` to invoice the order didn’t display the gift card code or gift card amount applied.
+
+<!--- MC-35838-->
+
+*  Merchants with multiple websites can now use REST to create and update products while preserving  image and image-role inheritance. Previously, when a merchant used REST to create and update products, and a product was updated for store view, the default image roles were loaded and saved for that store view. As a result, the store-view image roles stopped inheriting from the default scope after update.
+
+<!--- MC-36419-->
+
+*  An unscoped integration user  account can now access a resource through REST when resource permissions allow access.
+
+<!--- MC-35975-->
+
+### Website Restrictions
+
+<!--- MC-32366-->
+
+*  Enabling Website Restriction no longer blocks Varnish ESI requests for customers logged in as guest. Previously, these guests could not access the home page main menu.
+
+<!--- MC-32369-->
+
+*  Magento now loads the product home page as expected when website restrictions are enabled.
+
+### Wishlist
+
+<!--- MC-35810-->
+
+*  You can now use the wishlist search feature to find a product in a public wish list in deployments where support for multiple wishlists is enabled. Previously, after a customer used the wishlist search to find a product, selecting it, and clicking **Add to cart**, Magento didn’t add the product to the cart and displayed this error: `Invalid Form Key. Please refresh the page`.
```

Comment:
Is "wishlist" one or two words? I see both variations in this paragraph.

Reply:
will change to "wish list" in both files. That's our glossary spelling, although industry standard is "wishlist'.